hysds/user_rules_job.py

```diff
@@ -141,12 +141,13 @@ def evaluate_user_rules_job(job_id, alias=STATUS_ALIAS):
 @backoff.on_exception(
     backoff.expo, socket.error, max_tries=backoff_max_tries, max_value=backoff_max_value
 )
-def queue_finished_job(_id):
+def queue_finished_job(_id, index=None):
```

Comment:
Should we default to JOB_STATUS_ALIAS?

```suggestion
def evaluate_user_rules_job(job_id, index=JOB_STATUS_ALIAS):
```

Reply:
@mcayanan: should the suggestion be:
```suggestion
def queue_finished_job(_id, index=JOB_STATUS_ALIAS):
```

Reply:
yes. sorry I had a typo in my suggestion.